packages/terra-application-links/tests/wdio/application-tabs-spec.js

```diff
@@ -72,4 +72,31 @@ describe('ApplicationTabs', () => {
 
     Terra.should.matchScreenshot('#test-tabs', { selector: '#test-tabs', viewports });
   });
+
+  describe('Displays a application tabs start aligned', () => {
+    beforeEach(() => {
+      browser.url('/#/raw/tests/application-links/application-tabs-start');
+      browser.pause(50);
+    });
+
+    Terra.should.matchScreenshot('#test-tabs', { selector: '#test-tabs', viewports });
```

Comment:
Since we are not taking multiple interaction screenshots within this context, we really don't need to add a screenshot name to this instance, not providing on would name it `Display_an_appliaiton_tabs_start_aligned[default]` vs providing '#test-tabs' which results in the screenshot name of `Display_an_appliaiton_tabs_start_aligned[#test-tabs]`.